### PR TITLE
JAVA-11495 Moved gradle modules to gradle-modules

### PR DESCRIPTION
**Pull request by dkapil**

### CODE REVIEW

**Comment by kwoyke** [comment]
retest this please

File: gradle-modules/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Akka
```
Comment (kwoyke) [inline gradle-modules/README.md:1]: Akka?

Comment (dkapil) [reply gradle-modules/README.md:1]: Damn! :) . Fixing it.
Copied this from the newly created akka-modules

File: gradle-modules/settings.gradle [hunk 2]
```diff
@@ -0,0 +1,3 @@
+rootProject.name = 'gradle-modules'
+include 'gradletutorial'
```
[the remark this answers is not in the document]
Comment (dkapil) [reply gradle-modules/settings.gradle:2]: gradletutorials is the one inside gradle-modules called "gradletutorials"
gradle-5-articles is "gradle-5"

these modules are not correctly named as per their folder name.
should we fix that in this jira?

[the remark this answers is not in the document]
Comment (dkapil) [reply gradle-modules/settings.gradle]: I have fixed their names now

**Pull request merged**